ansible_collections/arista/avd/molecule/cv_deploy/cc_false.yml

```diff
@@ -12,46 +12,54 @@
     structured_dir: "{{ playbook_dir }}/intended/structured_configs/test_configs"
     intended_tag_device: avd-ci-leaf1
     intended_tags: "{{ lookup('file', structured_dir ~ '/' ~ intended_tag_device ~ '.yml')| from_yaml }}"
+    test_id: "cc-run-false"
+    cv_common_pattern: "avd-cv-deploy-{{ test_id }}"
 
   tasks:
-    - name: Generate random string
+    - name: "{{ test_id | upper }} Banner"
+      run_once: true
+      ansible.builtin.debug:
+        msg:
+          - "######################################################################"
+          - "### STARTING MOLECULE TEST {{ '{:<38}'.format(test_id[:38]) | upper }} ####"
+          - "######################################################################"
+
+    - name: "{{ test_id | upper }} Generate random string"
+      run_once: true
       ansible.builtin.set_fact:
         r: "{{ lookup('password', '/dev/null chars=ascii_lowercase,digits length=4') }}"
-      run_once: true
 
-    - name: Set facts
+    - name: "{{ test_id | upper }} Set deploy facts"
+      run_once: true
       ansible.builtin.set_fact:
```

Comment:
Since you are now changing it completely to a pattern, we don't have to have this set_fact task at all. Just give the inputs on the cv_deploy task itself. Fewer tasks makes it easier to follow.

Reply:
Addressed in `a9043ed8d Remove separate set_facts tasks`